Revamp the ASM blocking page configuration section to include in 2 pages (#16360)
DataDog/documentation: layouts/shortcodes/asm-protection-page-configuration.md

The blocked requests feature JSON or HTML content. If the [`Accept` HTTP header][19] is pointing to HTML—like `text/html`—, the HTML content is used; otherwise, the JSON one is used. 

Both sets of content is embedded in the Datadog Tracer library package and loaded locally. See examples of the templates for [HTML][101] and [JSON][102] in the Datadog Java tracer source code on GitHub.

The HTML and JSON content can both be changed using the `DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML` and `DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON` environment variables. Alternatively, you can use the `dd.appsec.http.blocked.template.html` or `dd.appsec.http.blocked.template.json` configuration entries.

> < img src="/security/application_security/asm-blocking-page-html.png" alt="The page displayed as ASM blocks requests originating from blocked IPs" width="75%" style="center" >

Reviewer: ```suggestion
<img src="/security/application_security/asm-blocking-page-html.png" alt="The page displayed as ASM blocks requests originating from blocked IPs" width="75%" style="center">
```

> The blocked requests feature JSON or HTML content. If the [`Accept` HTTP header][19] is pointing to HTML - like `text/html` -, the HTML content is used, otherwise the JSON one is. 

Reviewer: ```suggestion
The blocked requests feature JSON or HTML content. If the [`Accept` HTTP header][19] is pointing to HTML—like `text/html`—, the HTML content is used; otherwise, the JSON one is used. 
```

> Both sets of content is embedded in the Datadog tracer library package and loaded locally. See the examples of the templates for [HTML][17] and [JSON][18] in the Datadog Java tracer source code on Github.

Reviewer: ```suggestion
Both sets of content is embedded in the Datadog Tracer library package and loaded locally. See examples of the templates for [HTML][101] and [JSON][102] in the Datadog Java tracer source code on GitHub.
```

> The HTML and JSON content can both be changed using the `DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML` and `DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON` environment variables. Alternatively, you can use the `dd.appsec.http.blocked.template.html` / `dd.appsec.http.blocked.template.json` configuration entries.

Reviewer: ```suggestion
The HTML and JSON content can both be changed using the `DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML` and `DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON` environment variables. Alternatively, you can use the `dd.appsec.http.blocked.template.html` or `dd.appsec.http.blocked.template.json` configuration entries.
```

> {{< img src="/security/application_security/asm-blocking-page-html.png" alt="The page displayed as ASM blocks requests originating from blocked IPs" width="75%">}}

Reviewer: ```suggestion
< img src="/security/application_security/asm-blocking-page-html.png" alt="The page displayed as ASM blocks requests originating from blocked IPs" width="75%" style="center" >

[101]: https://github.com/DataDog/dd-trace-java/blob/master/dd-java-agent/agent-bootstrap/src/main/resources/datadog/trace/bootstrap/blocking/template.html
[102]: https://github.com/DataDog/dd-trace-java/blob/master/dd-java-agent/agent-bootstrap/src/main/resources/datadog/trace/bootstrap/blocking/template.json
```
